schemas/com.snowplowanalytics.snowplow.media/ad/jsonschema/1-0-0

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity with information about the currently played ad",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.media",
+    "name": "ad",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Friendly name of the ad.",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "adId": {
+      "type": "string",
+      "maxLength": 256,
+      "description": "Unique identifier for the ad."
+    },
+    "creativeId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 4096,
+      "description": "The ID of the ad creative."
+    },
+    "podPosition": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "The position of the ad within the ad break, starting with 1.",
+      "minimum": 0,
```

Comment:
This was probably covered, did we intend `minimum: 1`/`exclusiveMinimum: true` here to match the description?

Reply:
I do remember changing this already based on some comment, must have been in a different place... Have changed to minimum 1.